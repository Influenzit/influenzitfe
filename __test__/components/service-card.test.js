import renderer from 'react-test-renderer';
import ServiceCard from '../../components/service-card'
import { Provider } from 'react-redux';
import configureMockStore from "redux-mock-store";
import { QueryClientProvider, QueryClient } from "@tanstack/react-query";
import React from 'react';
jest.mock('next/dist/client/router', () => require('next-router-mock'));
const mockStore = configureMockStore();
const store = mockStore({
    status: {
        loading: false,
        success: false,
        error: false,
        message: "",
        userType: "",
        currentConversation: 0,
        showSidebar: false,
        showLogoutModal: false,
    }
});

it('renders correctly', () => {
  const queryClient = new QueryClient();
  const tree = renderer
    .create(
        <QueryClientProvider client={queryClient}>
            <Provider store={store}>
                <ServiceCard />
            </Provider>
        </QueryClientProvider>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
