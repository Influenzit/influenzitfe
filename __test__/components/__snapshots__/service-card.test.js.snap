// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-feb2d95d-0 ekyJDo"
>
  <button
    onClick={[Function]}
  >
    <span
      style={
        {
          "background": "none",
          "border": 0,
          "boxSizing": "border-box",
          "display": "inline-block",
          "height": "initial",
          "margin": 0,
          "maxWidth": "100%",
          "opacity": 1,
          "overflow": "hidden",
          "padding": 0,
          "position": "relative",
          "width": "initial",
        }
      }
    >
      <span
        style={
          {
            "background": "none",
            "border": 0,
            "boxSizing": "border-box",
            "display": "block",
            "height": "initial",
            "margin": 0,
            "maxWidth": "100%",
            "opacity": 1,
            "padding": 0,
            "width": "initial",
          }
        }
      >
        <img
          alt=""
          aria-hidden={true}
          src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2720%27%20height=%2720%27/%3e"
          style={
            {
              "background": "none",
              "border": 0,
              "display": "block",
              "height": "initial",
              "margin": 0,
              "maxWidth": "100%",
              "opacity": 1,
              "padding": 0,
              "width": "initial",
            }
          }
        />
      </span>
      <img
        alt=""
        data-nimg="intrinsic"
        decoding="async"
        onError={[Function]}
        onLoad={[Function]}
        src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
        style={
          {
            "border": "none",
            "bottom": 0,
            "boxSizing": "border-box",
            "display": "block",
            "height": 0,
            "left": 0,
            "margin": "auto",
            "maxHeight": "100%",
            "maxWidth": "100%",
            "minHeight": "100%",
            "minWidth": "100%",
            "objectFit": undefined,
            "objectPosition": undefined,
            "padding": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
            "width": 0,
          }
        }
      />
      <noscript>
        <img
          alt=""
          data-nimg="intrinsic"
          decoding="async"
          loading="lazy"
          src="/bookmark.svg"
          srcSet="/bookmark.svg 1x, /bookmark.svg 2x"
          style={
            {
              "border": "none",
              "bottom": 0,
              "boxSizing": "border-box",
              "display": "block",
              "height": 0,
              "left": 0,
              "margin": "auto",
              "maxHeight": "100%",
              "maxWidth": "100%",
              "minHeight": "100%",
              "minWidth": "100%",
              "objectFit": undefined,
              "objectPosition": undefined,
              "padding": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
              "width": 0,
            }
          }
        />
      </noscript>
    </span>
  </button>
  <div
    className="sc-feb2d95d-3 ceJcdJ"
    onClick={[Function]}
  >
    <span
      style={
        {
          "background": "none",
          "border": 0,
          "bottom": 0,
          "boxSizing": "border-box",
          "display": "block",
          "height": "initial",
          "left": 0,
          "margin": 0,
          "opacity": 1,
          "overflow": "hidden",
          "padding": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
          "width": "initial",
        }
      }
    >
      <img
        alt=""
        data-nimg="fill"
        decoding="async"
        onError={[Function]}
        onLoad={[Function]}
        src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
        style={
          {
            "border": "none",
            "bottom": 0,
            "boxSizing": "border-box",
            "display": "block",
            "height": 0,
            "left": 0,
            "margin": "auto",
            "maxHeight": "100%",
            "maxWidth": "100%",
            "minHeight": "100%",
            "minWidth": "100%",
            "objectFit": "cover",
            "objectPosition": "center",
            "padding": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
            "width": 0,
          }
        }
      />
      <noscript>
        <img
          alt=""
          data-nimg="fill"
          decoding="async"
          loading="lazy"
          src=""
          style={
            {
              "border": "none",
              "bottom": 0,
              "boxSizing": "border-box",
              "display": "block",
              "height": 0,
              "left": 0,
              "margin": "auto",
              "maxHeight": "100%",
              "maxWidth": "100%",
              "minHeight": "100%",
              "minWidth": "100%",
              "objectFit": "cover",
              "objectPosition": "center",
              "padding": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
              "width": 0,
            }
          }
        />
      </noscript>
    </span>
  </div>
  <div
    className="sc-feb2d95d-4 jZUrDU"
    onClick={[Function]}
  >
    <div
      className="sc-feb2d95d-5 cetdlH"
    >
      <div>
        <span
          style={
            {
              "background": "none",
              "border": 0,
              "boxSizing": "border-box",
              "display": "inline-block",
              "height": "initial",
              "margin": 0,
              "maxWidth": "100%",
              "opacity": 1,
              "overflow": "hidden",
              "padding": 0,
              "position": "relative",
              "width": "initial",
            }
          }
        >
          <span
            style={
              {
                "background": "none",
                "border": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": "initial",
                "margin": 0,
                "maxWidth": "100%",
                "opacity": 1,
                "padding": 0,
                "width": "initial",
              }
            }
          >
            <img
              alt=""
              aria-hidden={true}
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2712%27%20height=%2712%27/%3e"
              style={
                {
                  "background": "none",
                  "border": 0,
                  "display": "block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "padding": 0,
                  "width": "initial",
                }
              }
            />
          </span>
          <img
            alt=""
            data-nimg="intrinsic"
            decoding="async"
            onError={[Function]}
            onLoad={[Function]}
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style={
              {
                "border": "none",
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": 0,
                "left": 0,
                "margin": "auto",
                "maxHeight": "100%",
                "maxWidth": "100%",
                "minHeight": "100%",
                "minWidth": "100%",
                "objectFit": undefined,
                "objectPosition": undefined,
                "padding": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 0,
              }
            }
          />
          <noscript>
            <img
              alt=""
              data-nimg="intrinsic"
              decoding="async"
              loading="lazy"
              src="/facebook-icon.svg"
              srcSet="/facebook-icon.svg 1x, /facebook-icon.svg 2x"
              style={
                {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
          </noscript>
        </span>
        <span
          style={
            {
              "background": "none",
              "border": 0,
              "boxSizing": "border-box",
              "display": "inline-block",
              "height": "initial",
              "margin": 0,
              "maxWidth": "100%",
              "opacity": 1,
              "overflow": "hidden",
              "padding": 0,
              "position": "relative",
              "width": "initial",
            }
          }
        >
          <span
            style={
              {
                "background": "none",
                "border": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": "initial",
                "margin": 0,
                "maxWidth": "100%",
                "opacity": 1,
                "padding": 0,
                "width": "initial",
              }
            }
          >
            <img
              alt=""
              aria-hidden={true}
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2712%27%20height=%2712%27/%3e"
              style={
                {
                  "background": "none",
                  "border": 0,
                  "display": "block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "padding": 0,
                  "width": "initial",
                }
              }
            />
          </span>
          <img
            alt=""
            data-nimg="intrinsic"
            decoding="async"
            onError={[Function]}
            onLoad={[Function]}
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style={
              {
                "border": "none",
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": 0,
                "left": 0,
                "margin": "auto",
                "maxHeight": "100%",
                "maxWidth": "100%",
                "minHeight": "100%",
                "minWidth": "100%",
                "objectFit": undefined,
                "objectPosition": undefined,
                "padding": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 0,
              }
            }
          />
          <noscript>
            <img
              alt=""
              data-nimg="intrinsic"
              decoding="async"
              loading="lazy"
              src="/instagram-icon.svg"
              srcSet="/instagram-icon.svg 1x, /instagram-icon.svg 2x"
              style={
                {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
          </noscript>
        </span>
        <span
          style={
            {
              "background": "none",
              "border": 0,
              "boxSizing": "border-box",
              "display": "inline-block",
              "height": "initial",
              "margin": 0,
              "maxWidth": "100%",
              "opacity": 1,
              "overflow": "hidden",
              "padding": 0,
              "position": "relative",
              "width": "initial",
            }
          }
        >
          <span
            style={
              {
                "background": "none",
                "border": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": "initial",
                "margin": 0,
                "maxWidth": "100%",
                "opacity": 1,
                "padding": 0,
                "width": "initial",
              }
            }
          >
            <img
              alt=""
              aria-hidden={true}
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2712%27%20height=%2712%27/%3e"
              style={
                {
                  "background": "none",
                  "border": 0,
                  "display": "block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "padding": 0,
                  "width": "initial",
                }
              }
            />
          </span>
          <img
            alt=""
            data-nimg="intrinsic"
            decoding="async"
            onError={[Function]}
            onLoad={[Function]}
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style={
              {
                "border": "none",
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": 0,
                "left": 0,
                "margin": "auto",
                "maxHeight": "100%",
                "maxWidth": "100%",
                "minHeight": "100%",
                "minWidth": "100%",
                "objectFit": undefined,
                "objectPosition": undefined,
                "padding": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 0,
              }
            }
          />
          <noscript>
            <img
              alt=""
              data-nimg="intrinsic"
              decoding="async"
              loading="lazy"
              src="/twitter-icon.svg"
              srcSet="/twitter-icon.svg 1x, /twitter-icon.svg 2x"
              style={
                {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
          </noscript>
        </span>
        <span
          style={
            {
              "background": "none",
              "border": 0,
              "boxSizing": "border-box",
              "display": "inline-block",
              "height": "initial",
              "margin": 0,
              "maxWidth": "100%",
              "opacity": 1,
              "overflow": "hidden",
              "padding": 0,
              "position": "relative",
              "width": "initial",
            }
          }
        >
          <span
            style={
              {
                "background": "none",
                "border": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": "initial",
                "margin": 0,
                "maxWidth": "100%",
                "opacity": 1,
                "padding": 0,
                "width": "initial",
              }
            }
          >
            <img
              alt=""
              aria-hidden={true}
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2712%27%20height=%2712%27/%3e"
              style={
                {
                  "background": "none",
                  "border": 0,
                  "display": "block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "padding": 0,
                  "width": "initial",
                }
              }
            />
          </span>
          <img
            alt=""
            data-nimg="intrinsic"
            decoding="async"
            onError={[Function]}
            onLoad={[Function]}
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style={
              {
                "border": "none",
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": 0,
                "left": 0,
                "margin": "auto",
                "maxHeight": "100%",
                "maxWidth": "100%",
                "minHeight": "100%",
                "minWidth": "100%",
                "objectFit": undefined,
                "objectPosition": undefined,
                "padding": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 0,
              }
            }
          />
          <noscript>
            <img
              alt=""
              data-nimg="intrinsic"
              decoding="async"
              loading="lazy"
              src="/tiktok-icon.svg"
              srcSet="/tiktok-icon.svg 1x, /tiktok-icon.svg 2x"
              style={
                {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
          </noscript>
        </span>
        <span
          style={
            {
              "background": "none",
              "border": 0,
              "boxSizing": "border-box",
              "display": "inline-block",
              "height": "initial",
              "margin": 0,
              "maxWidth": "100%",
              "opacity": 1,
              "overflow": "hidden",
              "padding": 0,
              "position": "relative",
              "width": "initial",
            }
          }
        >
          <span
            style={
              {
                "background": "none",
                "border": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": "initial",
                "margin": 0,
                "maxWidth": "100%",
                "opacity": 1,
                "padding": 0,
                "width": "initial",
              }
            }
          >
            <img
              alt=""
              aria-hidden={true}
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2712%27%20height=%2712%27/%3e"
              style={
                {
                  "background": "none",
                  "border": 0,
                  "display": "block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "padding": 0,
                  "width": "initial",
                }
              }
            />
          </span>
          <img
            alt=""
            data-nimg="intrinsic"
            decoding="async"
            onError={[Function]}
            onLoad={[Function]}
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style={
              {
                "border": "none",
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": 0,
                "left": 0,
                "margin": "auto",
                "maxHeight": "100%",
                "maxWidth": "100%",
                "minHeight": "100%",
                "minWidth": "100%",
                "objectFit": undefined,
                "objectPosition": undefined,
                "padding": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 0,
              }
            }
          />
          <noscript>
            <img
              alt=""
              data-nimg="intrinsic"
              decoding="async"
              loading="lazy"
              src="/youtube-icon.svg"
              srcSet="/youtube-icon.svg 1x, /youtube-icon.svg 2x"
              style={
                {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
          </noscript>
        </span>
      </div>
      <div>
        <span />
      </div>
    </div>
    <p />
    <div
      className="sc-feb2d95d-6 iNDdWk"
    >
      <div>
        <span
          style={
            {
              "background": "none",
              "border": 0,
              "bottom": 0,
              "boxSizing": "border-box",
              "display": "block",
              "height": "initial",
              "left": 0,
              "margin": 0,
              "opacity": 1,
              "overflow": "hidden",
              "padding": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
              "width": "initial",
            }
          }
        >
          <img
            alt=""
            data-nimg="fill"
            decoding="async"
            onError={[Function]}
            onLoad={[Function]}
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style={
              {
                "border": "none",
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": 0,
                "left": 0,
                "margin": "auto",
                "maxHeight": "100%",
                "maxWidth": "100%",
                "minHeight": "100%",
                "minWidth": "100%",
                "objectFit": "cover",
                "objectPosition": "center",
                "padding": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 0,
              }
            }
          />
          <noscript>
            <img
              alt=""
              data-nimg="fill"
              decoding="async"
              loading="lazy"
              src=""
              style={
                {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": "cover",
                  "objectPosition": "center",
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
          </noscript>
        </span>
      </div>
      <p />
    </div>
  </div>
</div>
`;
